src/set_theory/cardinal/basic.lean

```diff
@@ -225,35 +227,38 @@ mk_subtype_le s
 theorem out_embedding {c c' : cardinal} : c ≤ c' ↔ nonempty (c.out ↪ c'.out) :=
 by { transitivity _, rw [←quotient.out_eq c, ←quotient.out_eq c'], refl }
 
-theorem lift_mk_le {α : Type u} {β : Type v} :
+/-- The canonical way to compare a cardinal `a : cardinal.{u}` with a cardinal `b : cardinal.{v}` is
+to compare `cardinal.lift.{v} a` with `cardinal.lift.{u} b`.
+
+It might be tempting to be more general and compare `cardinal.lift.{max v w} a` with
+`cardinal.lift.{max u w} b`. However, given the lemmas `cardinal.lift_umax_le`,
+`cardinal.lift_umax_lt`, and `cardinal.lift_umax_eq`, this small generalization is not worth the
+elaboration problems it causes elsewhere. -/
+library_note "cardinal comparison in different universes"
+
+/- This is the sole exception to the rule established previously, as it's used to prove
```

Comment:
```suggestion
/- This is the sole exception to the rule established previously in note [cardinal comparison in different universes], as it's used to prove
```
(with line wrapping), as library note don't show up as adjacent in the docs

Reply:
This is a private theorem (for the outlined reason) so this is probably not too relevant. Doc has been updated however.